src/App.js

```diff
@@ -24,6 +25,7 @@ function App() {
             path="/:category_name"
             element={<Category category={category} />}
           ></Route>
+          <Route path="/review-:review_id" element={<Review />} />
```

Comment:
This isn't standard syntax Ben, this needs be something more conventional such as `/reviews/:review_id`